.buildkite/pipeline.yml

```diff
@@ -123,6 +125,10 @@ steps:
         agents:
           queue: opensource
         plugins:
+          test-collector#v1.10.2:
+            files: "reports/TEST-*.xml"
+            format: "junit"
+            branch: "^main|next$$"
```

Comment:
Again, apologies, I could have picked up on this before - I would put this after the docker-compose plugin.  It's probably safe here as the plugins are run by the relevant hook and only in the order they appear if in the same hook, but conceptually it happens after Maze Runner runs and so I think it would be easier to understand and maintain.